capstonebundle/plugin/mos65xx/mos65xx.cpp

```diff
@@ -0,0 +1,35 @@
+// mos65xx.cpp
+#include "mos65xx.h"
+
+MOS65XX::MOS65XX(RDContext* ctx, cs_mode mode): Capstone(ctx, CS_ARCH_MOS65XX, mode) { }
+
+void MOS65XX::emulate(RDEmulateResult* result)
+{
+    rd_address address = RDEmulateResult_GetAddress(result);
+    auto* insn = this->decode(address, RDEmulateResult_GetView(result));
+    if(!insn) return;
+
+    // Instruction is decoded, you can use Capstone API to analyze it
+    
+    RDContext_SetAddressAssembler(m_context, address, this->endianness() == Endianness_Big ? MOS65XXBE_ID : MOS65XXLE_ID);
+    if(!this->decode(address, RDEmulateResult_GetView(result))) return;
+
+    RDEmulateResult_SetSize(result, m_insn->size);
+
+}
+
+void MOS65XX::render(const RDRendererParams* rp)
+{
+  // You can render instructions here
+  auto* insn = this->decode(rp->address, &rp->view);
```

Comment:
Try to show the instruction's mnemonic with `RDRender_MnemonicWord` API: 

Like here: https://github.com/REDasmOrg/REDasm-Assemblers/blob/db0d698029d6776f4b9f3612498439621b473987/capstonebundle/plugin/arm32/common.cpp#L68